tests/cases/compiler/classImplementsPrimitive.ts

```diff
@@ -0,0 +1,17 @@
+// classes cannot implement primitives
+
+class C implements number { }
+class C2 implements string { }
+class C3 implements boolean { }
+class C4 implements Void  { }
+class C4a implements void {}
+class C5 implements Null { }
+class C5a implements null { }
+class C6 implements undefined { }

```

Comment:
testing null and undefined doesn't feel relevant to this PR since the new errors don't apply. Can you grep for existing tests that have 'implements null/undefined'? If there aren't any, feel free to leave them here; otherwise, remove them.

Reply:
There aren't both. However, I would remove these cases because these tests aren't relevant to this PR as you said.